src/replication.thrift

```diff
@@ -478,6 +478,28 @@ struct query_disk_info_response
     4:list<disk_info> disk_infos;
 }
 
+// This request is sent from client to replica_server.
+struct replica_disk_migrate_request
+{
+    1:dsn.gpid pid
+    2:string origin_disk;
+    3:string target_disk;
+}
+
+// This response is from replica_server to client.
+struct replica_disk_migrate_response
+{
+   // Possible error:
+   // -ERR_OK: start do replica disk migrate
+   // -ERR_BUSY: current replica migration is running
+   // -ERR_INVALID_STATE: current replica partition status isn't secondary
+   // -ERR_INVALID_PARAMETERS: origin disk is equal with target disk
+   // -ERR_OBJECT_NOT_FOUND: origin or target disk isn't existed, origin disk doesn't exist current replica
+   // -ERR_PATH_ALREADY_EXIST: target disk has existed current replica
```

Comment:
```suggestion
   // -ERR_PATH_ALREADY_EXIST: target disk has existed current replica
```

I think disk migration should proceed on, even if the replica exists on the target disk.
This is a rare case though. But given this condition, it means the previous disk migration must have failed for some reason.

Then the replica on the target disk should be seen as an abandoned one, which may contain data fault. 

If this happens, the disk migration task can automatically remove the replica directory, and continue execution without interference.

Reply:
`ERR_PATH_ALREADY_EXIST` means the disk info stored in `dir_node` object, if previous task failed, the garbage will be deleted and start new task:
https://github.com/XiaoMi/rdsn/blob/854cff67948f060d6d87a209bde722873b703020/src/replica/disk_migration/replica_disk_migrator.cpp#L205-L212